### PR TITLE
Ucontroller

### PR DESCRIPTION
**Pull request by tsuberim**
Support for UController.

### CODE REVIEW

File: ops/config.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ common: &common
 mainnet:
   <<: *common
   addresses:
-    GenesisProtocol: "0x8940442e7f54e875c8c1c80213a4aee7eee4781c"
+    Reputation: "0x13ad61aa8f695ce64711a51a6feccb48a275aaf8"
```
Comment (orenyodfat) [inline ops/config.yaml]: Is this related to this PR ? UController ?

File: src/mappings/UController.graphql [hunk 2]
```diff
@@ -0,0 +1,63 @@
+type UControllerOrganization @entity {
+    avatarAddress: ID!
```
Comment (orenyodfat) [inline src/mappings/UController.graphql]: how about 'avatarAddress : Avatar! '?

File: src/mappings/UController.graphql [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    nativeToken: Bytes!
```
Comment (orenyodfat) [inline src/mappings/UController.graphql]: nativeToken: NativeToken!

File: src/mappings/UController.graphql (hunk 2 above)
Comment (tsuberim) [reply src/mappings/UController.graphql]: Why do we need another entity? also Avatar is another contract.

File: src/mappings/UController.graphql (hunk 3 above)
Comment (tsuberim) [reply src/mappings/UController.graphql]: Why do we need another entity? also ERC20 is another contract.

File: ops/config.yaml (hunk 1 above)
Comment (tsuberim) [reply ops/config.yaml]: not related but needed.

Comment (orenyodfat) [reply ops/config.yaml]: how it is needed ? 
lets do other config.yaml for the reputation..

File: src/mappings/UController.graphql (hunk 2 above)
Comment (orenyodfat) [reply src/mappings/UController.graphql]: this is the power of Graph(ql).
controller point to Avatar

Comment (tsuberim) [reply src/mappings/UController.graphql]: But we have no Avatar entity right now

Comment (orenyodfat) [reply src/mappings/UController.graphql]: ok. lets do it only for rep
